%!TEX root = ../csustudy_main.tex
\keywordsen{Gene regulatory networks; single-cell RNA-seq data; mutual information; dynamic network construction; cellular heterogeneity}
\categoryen{TP391}
\itemcountcn{There are \totalfigures\ figures, \totaltables\ tables, and \total{citnum}\ citations in this study.}
% \addcontentsline{toc}{section}{ABSTRACT}
\begin{abstracten}\setlength{\baselineskip}{20pt}

% Note: TODO in the final runs!

Gene regulatory networks (GRNs) are formed by the complex regulatory relationships among genes. 
The construction of GRNs helps us to understand the mechanism of gene regulation, the structural and functional information of biological network systems, and also the mechanisms of complex diseases such as tumorigenesis at the molecular level.
The construction of gene regulatory network is one of the core and most important problems in systems biology.
In this study, effective methods for gene regulatory network construction are proposed on datasets generated by  DNA microarray and single-cell  RNA-seq  sequencing technologies, 
to address  the difficulties in the study of gene regulatory network inference, such as low accuracy and sparse network.
The innovations and main work in this study are as follows:

(1) To address the problem of low accuracy of gene network construction based on mutual information, 
a gene regulatory network construction method based on mutual information and local structure, Loc-PCA-CMI, is proposed in this study. 
The method identifies locally overlapping gene clusters based on their co-expression relationships, and then infer the local network structure of each gene cluster based on the path consistency algorithm of conditional mutual information (PCA-CMI), 
and finally constructs the final gene regulatory network by aggregating the local network structure.
We evaluate Loc-PCA-CMI on the DREAM3 knockout dataset and compare its performance with four other information theory-based methods for inferring network structure.
In particular, the experimental results on the DREAM3 dataset show that Loc-PCA-CMI  reduces the redundant dependencies 
in the construction  and outperforms the other four  methods in terms of AUPR.

(2) To address the shortcomings that data-driven methods cannot construct global networks, 
a data-driven and sampling-based gene regulatory network construction method, D3GRN, 
is proposed in this study. 
The method transforms the regulatory relationship for each target gene into a functional decomposition problem, 
and uses an improved data-driven method ARNI to infer each sub-network.
Experimental results on the DREAM4 and DREAM5 benchmark datasets show that D3GRN outperforms the other three methods in terms of AUPR.


(3) To address the problem that gene regulatory networks related to cell type and cell activity cannot be inferred simultaneously from single-cell RNA-seq data, 
a matrix decomposition-based gene regulatory network construction method, scGRNHunter, is proposed. 
The cell type program and the cell activity program are also isolated, and then a gene regulatory network based on each program is constructed in conjunction with the public database TRRUST. 
The experimental results on the publicly available brain-like organ single-cell RNA-Seq dataset show that our proposed scGRNHunter method can effectively infer the identity and activity of subprograms, 
and construct cell type-based and cell activity-based gene regulatory networks based on them.

% (3) In order to solve the problem of inaccurate cell clustering on single-cell RNA-seq datasets, 
% RafClust is proposed as a random forest similarity learning-based single-cell clustering method, 
% which uses multiple correlation measures to characterize cells, 
% and then uses a random forest regression model to further learn the cell-to-cell similarity matrix based on similarity. 
% The matrix is applied with hierarchical clustering to determine the final category of cells. 
% The experimental results on ten single-cell datasets show that RafClust outperforms the other six benchmark methods in terms of AUPR.

% (4) To address the problem of time-consuming and memory-consuming algorithms for identifying rare cells from single-cell RNA-seq data, 
% DoRC is proposed as an isolated forest-based single-cell rare cell identification method, 
% which uses isolated forest to efficiently generate a rarity fraction for each cell and combines it with a threshold method to binary label the cells for sparsity. 
% The experimental results on a single cell expression profile of 68k human blood cells from single cell RNA-seq data show that DoRC is highly efficient in classifying human blood dendritic cell subtypes. 
% In addition, DoRC can identify rare cells in simulated datasets and is sensitive to cell type characteristics.

(4) To address the current problem of inaccurate cell clustering on single-cell RNA-seq datasets, 
a single-cell clustering method based on random forest similarity learning named RafClust is proposed in this study.  
This method uses multiple correlation measures to characterize cells, and uses a random forest regression model to further learn the cell-to-cell similarity matrix, and then uses a hierarchical clustering based on the similarity matrix to determine the final class of cells. 
Hierarchical clustering is used to determine the final class of cells based on the similarity matrix. 
The experimental results on ten single-cell datasets show that RafClust outperformed the other six methods in terms of ARI. 
Based on RafClust, DoRC is further proposed to identify rare cells from ultra-large-scale single-cell RNA-seq data, 
which efficiently assigns a rarity score to each cell using isolated forests, combined with a threshold method for binary annotation of cell sparsity. 
Experimental results on ultra-large-scale single-cell RNA-seq data ${\sim}68$k single-cell expression profiles of human blood cells show that,
DoRC is outstandingly effective in delineating human blood dendritic cell subtypes with high execution efficiency. 
In addition, DoRC can identify rare cells inside the simulated dataset and is sensitive to cell type characteristics as well.


\end{abstracten}